image/roxctl/rhtap.Dockerfile

```
@@ -40,18 +40,14 @@ LABEL \
     com.redhat.component="rhacs-roxctl-container" \
     name="rhacs-roxctl-rhel8" \
     maintainer="Red Hat, Inc." \
+    upstream-repo="github.com/stackrox/stackrox" \
```

Tom, sorry for back and forth on this, I was just looking at

```bash
$ docker inspect registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:20ce2094786a250cbb32b1d776a9e41fcdc1f69860e01bbbd48374ac32dc3da5
```

and saw labels they use there.
They set `io.openshift.build.source-location` like this:

```
"io.openshift.build.source-location": "https://github.com/openshift/kube-rbac-proxy",
```

This seems to be coming from something called "Build" (unambiguous and easy to find, right?) - https://docs.openshift.com/online/pro/dev_guide/builds/build_output.html#output-image-labels

I'm not saying we should follow exactly what they do, but we could change couple things to be more similar
```suggestion
    source-location="https://github.com/stackrox/stackrox" \
```

---

```json
{
    "architecture": "x86_64",
    "build-date": "2023-10-13T11:58:23",
    "com.redhat.component": "rhacs-roxctl-container",
    "com.redhat.license_terms": "https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI",
    "description": "The CLI for ACS",
    "distribution-scope": "public",
    "io.buildah.version": "1.31.0",
    "io.k8s.description": "The CLI for ACS",
    "io.k8s.display-name": "roxctl",
    "io.openshift.expose-services": "",
    "io.openshift.tags": "rhacs,roxctl,stackrox",
    "maintainer": "Red Hat, Inc.",
    "name": "rhacs-roxctl-rhel8",
    "quay.expires-after": "13w",
    "release": "1072.1696517598",
    "summary": "The CLI for ACS",
    "upstream-repo": "github.com/stackrox/stackrox",
    "url": "https://access.redhat.com/containers/#/registry.access.redhat.com/ubi8-minimal/images/8.8-1072.1696517598",
    "vcs-ref": "befdf5d01d9a9855307749e4196d05d3a4c7cbf5",
    "vcs-type": "git",
    "vendor": "Red Hat, Inc.",
    "version": "8.8"
}
```

These are all labels set on the image. `url`, `version` could be updated as well. 

Let us move this discussion (and its resolution) to another Jira task, since we also started [this discussion](https://redhat-internal.slack.com/archives/C04PZ7H0VA8/p1697127151309229). I would actually expect RHTAP to set more labels for us like the repository.

---

That discussion feels like RHTAP won't give us solution tomorrow, whereas we can at least erase wrong values today. https://github.com/stackrox/stackrox/pull/8167